engine/src/main/java/org/terasology/rendering/dag/AbstractNode.java

```diff
@@ -40,17 +40,20 @@
     private RenderTaskListGenerator taskListGenerator; // TODO: investigate ways to remove nodes influence on taskList
     private boolean enabled = true;
 
-    protected FBO requiresFBO(FBOConfig fboConfig, BaseFBOsManager frameBuffersManager) {
+    public void initialise(Object initialData) { } // added here to avoid adding it to all other nodes.
```

Comment:
Thank you @flo for the comment. I'm not 100% sure I understood it. Are you suggesting to make a `NodeFactory<T, S>`, type S being the type of data required by the initialize method? Or are you suggesting to simply constrain the type passed to initialise() with some kind of interface?

My problem right now is that I haven't gone through enough Node refactoring to know if there will be a pattern in the usage of the initialize(Object) method. If there was of course I'd require it to accept only instances implementing an interface or extending a base class. But if I have to look just at the BufferClearingNode, for example, its initial data is very type-specific.

In this context I could perhaps add a TODO and review the situation when most or all nodes have been refactored.

Is there any reason to _not_ use Object as input type? Of course a node could potentially get the wrong object, but isn't that easily solved using `instanceof`?